scripts/tv-verify-all.sh

```
@@ -0,0 +1,243 @@
+CMD_LABEL="indicators rsi"
+timeout 30 bun run src/tradingview/index.ts indicators rsi > /tmp/tv-verify-V4.1.log 2>&1; EC=$?
+if [[ $EC -eq 0 ]] && rg -q "STD;" /tmp/tv-verify-V4.1.log && rg -qi "Relative Strength" /tmp/tv-verify-V4.1.log; then record V4.1 "$CMD_LABEL" PASS ""
+else record V4.1 "$CMD_LABEL" FAIL "exit=$EC"; fi
+
+if [[ $HAS_SESSION -eq 1 ]]; then
+  timeout 30 bun run src/tradingview/index.ts indicators --filter saved > /tmp/tv-verify-V4.2.log 2>&1; EC=$?
+  if [[ $EC -eq 0 ]]; then record V4.2 "indicators --filter saved" PASS ""
+  else record V4.2 "indicators --filter saved" FAIL "exit=$EC"; fi
+else record V4.2 "indicators --filter saved" SKIP "no session"; fi
+
+if ls ~/.genesis-tools/tradingview/cache/ 2>/dev/null | rg -q .; then record V4.3 "cache dir" PASS ""
```

Shellcheck (0.11.0)

[info] Use find instead of ls to better handle non-alphanumeric filenames.

(SC2012)